ext/nnstreamer/tensor_filter/tensor_filter_trix_engine.cc

```diff
@@ -0,0 +1,353 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (C) 2020 Samsung Electronics
+ * Copyright (C) 2020 Dongju Chae <dongju.chae@samsung.com>
+ */
+/**
+ * @file tensor_filter_trix_engine.cc
+ * @date 20 Jan 2020
```

Comment:
Need to update date.

Reply:
The date should be the original date. The files were just moved from npu-engine repo.

Reply:
Code in nnstreamer.git is LGPL 2.1 only. Update it.

Reply:
Updated, thanks.